micro-rdk/src/common/conn/viam.rs

```diff
@@ -418,11 +418,10 @@ where
     pub fn run_forever(&mut self) -> ! {
         #[cfg(feature = "esp32")]
         {
-            use esp_idf_svc::sys::{esp_task_wdt_config_t, CONFIG_FREERTOS_NUMBER_OF_CORES};
```

Comment:
don't change that for now it was handled in #457 which we reverted for now but are probably going to merge back in